docs/settings.rst

```diff
@@ -22,7 +22,6 @@ DATABASE_URL                            DATABASES                   auto w/ Dock
 DJANGO_ADMIN_URL                        n/a                         'admin/'                                       raises error
 DJANGO_DEBUG                            DEBUG                       True                                           False
 DJANGO_SECRET_KEY                       SECRET_KEY                  auto-generated                                 raises error
-DJANGO_SECURE_BROWSER_XSS_FILTER        SECURE_BROWSER_XSS_FILTER   n/a                                            True
```

Comment:
That was wrong actually, since there is no env variable for `DJANGO_SECURE_BROWSER_XSS_FILTER`...